cli/src/action/mod.rs

```diff
@@ -128,3 +139,48 @@ fn msg_from_io_error(err: IoError) -> String {
         _ => "Unknown I/O error".into(),
     }
 }
+
+// determine if a full file path or just a file name has been given for the private key
+fn determine_key_file_path(key: &str) -> Result<PathBuf, CliError> {
```

Comment:
We should take whatever the "best" code we have for this currently and make it a function in cylinder, as we probably have half a dozen different implementations currently.

Reply:
Fixed